<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyslogWidget</class>
 <widget class="QWidget" name="SyslogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>168</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Sans</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="horizontalSpacing">
    <number>15</number>
   </property>
   <property name="verticalSpacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>30</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblServer">
     <property name="text">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="cmbServer">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>localhost</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPort">
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <widget class="QSpinBox" name="spinPort">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>514</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>___</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="Line" name="line">
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblMessage">
     <property name="text">
      <string>Message</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cmbMessage">
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToMinimumContentsLengthWithIcon</enum>
       </property>
       <item>
        <property name="text">
         <string>Test: %TestName%; Status: %Status%; Reply: %Reply%</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnMessage">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../qHostMonitor.qrc">
         <normaloff>:/img/loupe.png</normaloff>:/img/loupe.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblSeverity">
     <property name="text">
      <string>Severity</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="cmbSeverity">
     <item>
      <property name="text">
       <string>Emergency</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Alert</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Critical</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Error</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Warning</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Notice</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Info</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debug</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblFacility">
     <property name="text">
      <string>Facility</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSpinBox" name="spinFacility">
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>___</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../qHostMonitor.qrc"/>
  <include location="../../qHostMonitor.qrc"/>
 </resources>
 <connections/>
</ui>
